src/server/gc_worker/gc_raftdb.rs

```diff
@@ -0,0 +1,78 @@
+// Copyright 2021 TiKV Project Authors. Licensed under Apache-2.0.
```

Comment:
I don't think it should be put into the gc_worker dir. GC worker is at the level of MVCC which is quite unrelated with raft log.

Reply:
I agree with you on it. Considering the option only works when using rocksdb as raft log storage, I think engine_rocks dir is the best place for it.